pkg/util/log/eventpb/telemetry.proto

```diff
@@ -71,3 +71,31 @@ message CapturedIndexUsageStats {
   bool is_unique = 10 [(gogoproto.jsontag) = ",omitempty"];
   bool is_inverted = 11 [(gogoproto.jsontag) = ",omitempty"];
 }
+
+// CreateChangefeedQuery
+message CreateChangefeedQuery {
+  CommonEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+  CommonJobEventDetails job = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
```

Comment:
Not sure we should embed job details.  We do have core style changefeeds -- and those don't have job record(s).  So, it's a bit strange to see it here.  Secondly, does the job detail proto give us interesting information?  I.e. is knowing job id interesting?  Personally, I don't think so.  Furthermore, we are using "Description" of the job event details to put in, basically a redacted changefeed statement.  Why don't we use `CommonSQLEventDetails` which already has  statement field (which gets redacted)?